qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -884,6 +897,12 @@ def program(self, program_id: str, refresh: bool = False) -> RuntimeProgram:
             RuntimeProgramNotFound: If the program does not exist.
             IBMRuntimeError: If the request failed.
         """
+        issue_deprecation_msg(
+            msg="Custom programs are being deprecated",
+            version="0.13.1",
+            remedy="Please use the provided primitives instead.",
+            period="1 month",
```

Comment:
We should point people to the migration guide. Also the deprecation period is technically slightly less than a month. So the message should look something like

`
Custom programs are being deprecated with qiskit-ibm-runtime 0.14.0 and will be removed on November 27, 2023. You can instead convert your custom programs to use Qiskit Runtime primitives with Quantum Serverless. Refer to the migration guide for instructions: https://qiskit-extensions.github.io/quantum-serverless/migration/migration_from_qiskit_runtime_programs.html
`